scribus/text/storytext.cpp

```diff
@@ -167,7 +168,7 @@ int StoryText::cursorPosition() const
 	return d->cursorPosition;
 }
 
-void StoryText::setCursorPosition(int pos, bool relative)
+void StoryText::setCursorPosition(int pos, bool relative) const
```

Comment:
This changes the cursor position, so it shouldn’t be const. Only methods that changes nothing in the class should be const.